source/Calamari.AzureAppService.Tests/AppServiceIntegrationTest.cs

```diff
@@ -83,12 +79,27 @@ public async Task Setup()
                                                                 });
 
             //create the resource group
-            SubscriptionResource = ArmClient.GetSubscriptionResource(SubscriptionResource.CreateResourceIdentifier(SubscriptionId));
-            var response = await SubscriptionResource
+            subscriptionResource = ArmClient.GetSubscriptionResource(SubscriptionResource.CreateResourceIdentifier(SubscriptionId));
+            ResourceGroupResource staticResourceGroupResource = await subscriptionResource.GetResourceGroupAsync("calamari-testing-static-rg", CancellationToken);
+            
+            WindowsAppServicePlanResource = await staticResourceGroupResource.GetAppServicePlanAsync("calamari-testing-static-win-plan", CancellationToken);
+            LinuxAppServicePlanResource = await staticResourceGroupResource.GetAppServicePlanAsync("calamari-testing-static-linux-plan", CancellationToken);
+            WindowsContainerAppServicePlanResource = await staticResourceGroupResource.GetAppServicePlanAsync("calamari-testing-static-container-win-plan", CancellationToken);
+            LinuxAppServicePlanResource = await staticResourceGroupResource.GetAppServicePlanAsync("calamari-testing-static-linux-plan", CancellationToken);
+        }
+
+        [SetUp]
+        public virtual async Task SetUp()
+        {
+            var name = $"{DateTime.UtcNow:yyyyMMdd}-{Guid.NewGuid():N}";
+            
+            TestContext.WriteLine($"Creating resource group {ResourceGroupName}");
+            
+            var response = await subscriptionResource
                                  .GetResourceGroups()
                                  .CreateOrUpdateAsync(WaitUntil.Completed,
-                                                      ResourceGroupName,
-                                                      new ResourceGroupData(new AzureLocation(ResourceGroupLocation))
+                                                      name,
+                                                      new ResourceGroupData(AzureLocation.AustraliaEast)
```

Comment:
This needs to be `AustraliaEast` as the static resource groups are in `AustraliaEast`